modules/protocol/inspircd.c

```diff
@@ -107,18 +112,62 @@ static bool has_svshold = false;
 static bool has_cloakingmod = false;
 static bool has_shun = false;
 static bool has_svstopic_topiclock = false;
+static bool has_bannegate = false;	// Contrib module. --Elizafox
 static int has_protocol = 0;
 
+/* NOTE: modifies argp and argl, but not str
+ * You must have allocated at least maxsize space in argp and argl!
+ */
```

Comment:
Might be worth adding a high-level summary of the function (including inputs and outputs) for clarity